hack/install-etcd.sh

```diff
@@ -1,4 +1,9 @@
 #!/bin/bash
+
+# This prevents creation of temporary directories inside of the /tmp. This
+# script don't need them and creating them under root user could lead to other
+# scripts failing down the road.
+OS_TMP_ENV_SET=install-etcd
```

Comment:
@stevekuznetsov WDYT about this fix?

While I'm still testing it, the path that was causing a failure is already passing.

The benefits of it:
- it's short
- it makes script not create directories which this script isn't used
- we don't need to modify `vagrant install-origin` at all (it's Ruby code and AFIU it's not developing now)